R/utilities-simulation.R

```diff
@@ -134,6 +134,14 @@ simulateModelOnCore <- function(simulation,
   return(simulationResult)
 }
 
+
+#' @title defaultMaxSimulationsPerCore
+#' @description Default value for a scale factor used in a parallel simulation.  The product of this scale factor and the number of allowable cores (allowedCores) sets the maximum number of simulations that may be run on one core.
+#' @export
+defaultMaxSimulationsPerCore <- 2

```

Comment:
Don't we have an environement where we can put those type of global variables?

Reply:
moved to reEnv$defaultMaxSimulationsPerCore